Tests/test_cffi.py

```diff
@@ -49,10 +49,10 @@ def setUp(self):
             self.skipTest("No cffi")
 
     def _test_get_access(self, im):
-        """ Do we get the same thing as the old pixel access """
+        # Do we get the same thing as the old pixel access
 
-        """ Using private interfaces, forcing a capi access and
-        a pyaccess for the same image """
+        # Using private interfaces, forcing a capi access and
+        # a pyaccess for the same image
```

Comment:
Fair enough, yes, that would seem to be the better stylistic option. I've corrected this.